cms/src/helpers/genomicVariants.js

```diff
@@ -64,7 +64,12 @@ const buildVariantId = ({
   }
 };
 
-export const addGenomicVariantsFromMaf = async (name, mafData, { filename, fileId }) => {
+const buildModelUrl = caseId => `${BASE_GDC_URL}/cases/${caseId}`;
+const buildMafUrl = fileId => `${BASE_GDC_URL}/files/${fileId}`;
+const buildSequenceUrl = caseId =>
+  `${BASE_GDC_URL}/repository?facetTab=files&filters=%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22cases.case_id%22%2C%22value%22%3A%5B%22${caseId}%22%5D%7D%7D%5D%7D&searchTableTab=files`;
```

Comment:
where did this nonsense come from? lol... maybe leave a comment to help understand it.

Reply:
Good point 😂 Will update ASAP